pkg/kubelet/cm/devicemanager/plugin/v1beta1/server.go

```diff
@@ -190,3 +201,20 @@ func (s *server) visitClients(visit func(r string, c Client)) {
 	}
 	s.mutex.Unlock()
 }
+
+func (s *server) Name() string {
+	return "device-plugin"
+}
+
+func (s *server) Check(_ *http.Request) error {
+	if s.serverMonitor {
+		return nil
+	}
+	return fmt.Errorf("device plugin registration gRPC server failed and no device plugins can register")
+}
+
+// setHealthy sets the health status of the gRPC server.
+// If the server starts successfully, true is set; otherwise, false is set.
+func (s *server) setHealthy(healthy bool) {
```

Comment:
I'd probably have had 2 different functions because boolean args tend to lower the readability:

```
func (s *server) setHealthy() {
    s.serverMonitor = true
}

func (s *server) setUnhealthy() {
    s.serverMonitor = false
}
```

Reply:
> I'd probably have had 2 different functions because boolean args tend to lower the readability

agree